akka-coordination/src/main/resources/reference.conf

```diff
@@ -0,0 +1,3 @@
+akka.lease {
+
+}
```

Comment:
Should the retry-timeout go in the lease config instead of in the singleton config?

Reply:
In that case it should not go in the singleton config either, right?

Reply:
you can have different configs for different singletons by ClusterSingletonManagerSettings.apply(config) or amend the default ClusterSingletonManagerSettings using `withX`

Reply:
I think the config should be in the specific lease config so I've just removed this